external/ag-website-shared/src/markdoc/tags/majorTable.ts

```diff
@@ -0,0 +1,11 @@
+import { component } from '@astrojs/markdoc/config';
+import type { Render } from '@astrojs/markdoc/config';
+import type { Config, Schema } from '@markdoc/markdoc';
+
+export const majorTable: Schema<Config, Render> = {
+    render: component('../../external/ag-website-shared/src/components/major-table/MajorTable.astro'),
+    attributes: {
+        library: { type: String, required: true, matches: ['grid', 'charts'] },
+        major: { type: Number, required: true },
```

Comment:
I think you missed out on `type`

```suggestion
        major: { type: Number, required: true },
        type: { type: String, required: true, matches: ['migration', 'archive'] },
```

Reply:
This is intentional as I'm going to move the docs archive improvements to separate ticket to get this into staging quicker 👍